.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# [Choice] Go version: 1, 1.16, 1.15
+ARG VARIANT=1
+FROM mcr.microsoft.com/vscode/devcontainers/go:0-${VARIANT}
+
+# [Option] Install Node.js
+ARG DEBIAN_FRONTEND=noninteractive
+ARG INSTALL_NODE="true"
+ARG NODE_VERSION="lts/*"
+RUN if [ "${INSTALL_NODE}" = "true" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi \
+    # Install the Azure CLI
+    && echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/azure-cli.list \
+    && curl -sL https://packages.microsoft.com/keys/microsoft.asc | apt-key add - 2>/dev/null \
+    && apt-get update \
+    && apt-get install -y azure-cli \
+    # Install Docker CLI
+    && apt-get install -y gnupg-agent software-properties-common \
+    && curl -fsSL https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]')/gpg | (OUT=$(apt-key add - 2>&1) || echo $OUT) \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce-cli \
+    # Install Python
+    && apt-get update \
+    && apt-get install -y python3.6 python3-pip \
+    # Clean up
+    && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts
+
+# Install CI Tools
+RUN apt-get update \
+    && apt-get install -y shellcheck \
```

Comment:
It's recommended to run `rm -rf /var/lib/apt/lists/*` after an `apt-get install` https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

Reply:
thanks, added!